.eslintrc.json

```diff
@@ -0,0 +1,17 @@
+{
+    "env": {
+        "commonjs": true,
+        "es6": true,
+        "node": true
+    },
+    "extends": "eslint:recommended",
+    "globals": {
+        "Atomics": "readonly",
+        "SharedArrayBuffer": "readonly"
+    },
+    "parserOptions": {
+        "ecmaVersion": 2018
+    },
+    "rules": {
+    }
+}
```

Comment:
Your text editor really doesn't like newlines at the end of files!

Reply:
Blame eslint for auto-generating it that way! I also noticed that `cylc.tmLanguage.json` doesn't have a newline at EOF... I admit I've only just looked up why that might be a bad thing. Should I push a change with an added newline or is it not necessary?

Reply:
Might as well fix it, linters will often fail on this.